vendor.conf

```diff
@@ -28,4 +28,4 @@ github.com/pmezard/go-difflib v1.0.0
 github.com/stretchr/testify v1.2.2
 
 github.com/rancher/norman 457c15b94acae52afb5290aa315452c7621d452a
-github.com/rancher/types ee87bbdceacff5c566f901bdc57eec8330e7ed13
+github.com/rancher/types 336da48888a366ef68b859845585dc1b359a3742 https://github.com/rmweir/types
```

Comment:
This is temporary for testing right ? Wouldn't think we want a personal repo on here.